### DISCUSSION TITLE
Is the Demo Broken?

### DISCUSSION THREAD

**Discussion by dagilleland** [payloadcms/payload#3971] [Q&A]
So, when I go and try the [demo](https://demo.payloadcms.com/admin/login?redirect=), I can't log in. I don't know if it's just broken, or if the demo allows the demo user to change their password (which makes it kinda flakey for a demo).

Anyone?

**Comment by denolfe** [comment]
Interesting, it does look like there is an issue with credentials. Thanks for the report, we'll take a look! It resets every hour either way.

**Comment by DanRibbens** [comment]
Hey @dagilleland,

Got it! https://github.com/payloadcms/public-demo/pull/31

Thanks so much for letting us know!
